### PR TITLE
Hamming approach

### PR DESCRIPTION
**Pull request by bobahop**
Approach for Hamming

### CODE REVIEW

File: exercises/practice/hamming/.approaches/introduction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Approach: `zip` with `count`
+
+```kotlin
+object Hamming {
+
+    fun compute(leftStrand: String, rightStrand: String) =
+        if (leftStrand.length != rightStrand.length)
+            throw IllegalArgumentException("left and right strands must be of equal length")
+        else
+            (leftStrand zip rightStrand).count { it.first != it.second}
```
Comment (ErikSchierboom) [inline exercises/practice/hamming/.approaches/introduction.md:15]:
```suggestion
            (leftStrand zip rightStrand).count { it.first != it.second }
```

File: exercises/practice/hamming/.approaches/zip-count/content.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# `zip` with `count`
+
+```kotlin
+object Hamming {
+
+    fun compute(leftStrand: String, rightStrand: String) =
+        if (leftStrand.length != rightStrand.length)
+            throw IllegalArgumentException("left and right strands must be of equal length")
+        else
+            (leftStrand zip rightStrand).count { it.first != it.second}
```
Comment (ErikSchierboom) [inline exercises/practice/hamming/.approaches/zip-count/content.md:10]:
```suggestion
            (leftStrand zip rightStrand).count { it.first != it.second }
```

File: exercises/practice/hamming/.approaches/zip-count/snippet.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+object Hamming {
+
+    fun compute(leftStrand: String, rightStrand: String) =
+        if (leftStrand.length != rightStrand.length)
+            throw IllegalArgumentException("left and right strands must be of equal length")
+        else
+            (leftStrand zip rightStrand).count { it.first != it.second}
```
Comment (ErikSchierboom) [inline exercises/practice/hamming/.approaches/zip-count/snippet.txt:7]:
```suggestion
            (leftStrand zip rightStrand).count { it.first != it.second }
```

**Review by ErikSchierboom — APPROVED** [review]
Teeny nit

**Comment by bobahop** [comment]
@ErikSchierboom: All changes accepted.

**Pull request merged**